api/bags_api/src/test/scala/uk/ac/wellcome/platform/storage/bags/api/BagsApiFeatureTest.scala

```diff
@@ -128,6 +134,11 @@ class BagsApiFeatureTest
                 assertJsonStringsAreEqual(actualJson, expectedJson)
               }
 
+              val header: ETag = response.header[ETag].get
+              val etagValue = header.etag.value.replace("\"", "")
+
+              etagValue shouldBe storageManifest.idWithVersion
```

Comment:
I think this should be a little more explicit in what it expects – it’s just repeating the original code.

Maybe

```
etagValue shouldBe s"${storageManifest.space}/${storageManifest.externalIdentifier}/${storageManifest.version}"
```